### PR TITLE
Newcontrols

### PR DESCRIPTION
**Pull request by kwendel**
Have a look at our new controls. Moving the graph is now depending on the zoomlevel
Controls:
- and + for zooming
left and right arrow for moving the graph

### CODE REVIEW

File: src/main/java/tudelft/ti2806/pl3/Application.java [hunk 1]
```diff
@@ -3,19 +3,18 @@
...
 import tudelft.ti2806.pl3.zoomBar.ZoomBarController;
 
-import java.awt.Component;
-import java.io.File;
-import java.io.FileNotFoundException;
-
 import javax.swing.JFrame;
 import javax.swing.JLayeredPane;
 import javax.swing.JOptionPane;
+import java.awt.Component;
```
Comment (borismattijssen) [inline src/main/java/tudelft/ti2806/pl3/Application.java]: checkstyle error? wrong lexographical order

File: src/main/java/tudelft/ti2806/pl3/Application.java [hunk 2]
```diff
@@ -3,19 +3,18 @@
 import tudelft.ti2806.pl3.controls.KeyController;
 import tudelft.ti2806.pl3.controls.WindowController;
 import tudelft.ti2806.pl3.data.graph.GraphDataRepository;
-import tudelft.ti2806.pl3.visualization.GraphController;
 import tudelft.ti2806.pl3.sidebar.SideBarController;
```
Comment (borismattijssen) [inline src/main/java/tudelft/ti2806/pl3/Application.java]: checkstyle error

**Comment by TomBrouws** [comment]
Reported some Findbugs errors to Kasper

**Comment by TomBrouws** [comment]
Looks good now!

**Pull request merged**